runtime/doc/autocmd.txt

```diff
@@ -294,8 +294,8 @@ Name			triggered by ~
 |SourceCmd|		before sourcing a Vim script |Cmd-event|
 
 |VimResized|		after the Vim window size changed
-|VimResumed|		after Vim is resumed
-|VimSuspendPre|		before Vim is suspended
+|VimResume|		after nvim is resumed
+|VimSuspend|		before nvim is suspended
```

Comment:
in the docs we usually title-case `Nvim` unless referring to the command `nvim`.